bluemira/utilities/plot_tools.py

```diff
@@ -123,6 +123,43 @@ def make_gif(
     imageio.mimsave(Path(folder, f"{figname}.gif"), images, "GIF-FI", **kwargs)
 
 
+def xz_plot_setup(
+    pname,
+    folder,
+    save=False,
+    split_psi_plots: Optional[bool] = False,
+) -> Dict:
+    """Set up for an xz plot (poloidal slice)."""
+    if folder is None:
+        folder = try_get_bluemira_path(
+            "", subfolder="generated_data", allow_missing=not save
+        )
+
+    if split_psi_plots:
+        f, ax = plt.subplots(nrows=1, ncols=2, sharex=True, sharey=True)
+        ax[0].set_xlabel("$x$ [m]")
+        ax[0].set_ylabel("$z$ [m]")
+        ax[0].set_title("Coilset")
+        ax[0].set_aspect("equal")
+        ax[1].set_xlabel("$x$ [m]")
+        ax[1].set_ylabel("$z$ [m]")
+        ax[1].set_title("Plasma")
+        ax[1].set_aspect("equal")
+
+    else:
+        f, ax = plt.subplots()
+        ax.set_xlabel("$x$ [m]")
+        ax.set_ylabel("$z$ [m]")
+        ax.set_aspect("equal")
+    return {
+        "f": f,
+        "ax": ax,
+        "pname": pname,
+        "folder": folder,
+        "save": save,
+    }
+
+
 def save_figure(
     fig, name, save=False, folder=None, dpi=600, file_format="png", **kwargs
```

Comment:
Just had a quick look in passing and noticed this.
If possible could we change the file_format default to 'svg' here? 
It means that the figures will be vector graphics by default and therefore not pixelate on resizing. It does make having a default dpi a bit irrelevant but if it doesnt break anything leave that as is.